File: CódigosenC/PositivoNegativo.c
#include <stdio.h>
int n;
void main(){
    printf("Ingresa un Numero;\n");
    scanf("%i", &n);
    if(n!=0) n=(n>0) ? ( printf("%i Es positivo", n)) : ( printf("%i Es negativo", n));
    else printf("Es neutro");
}

File: Código_JS/positivoNegativo.js
//Determinar si un numero entero es positivo, negativo o neutro 

const prompt = require('prompt-sync')();

const numero = parseInt(prompt("Ingresa un número -> "));

if (numero > 0) {
  console.log("El número es positivo.");
}else if (numero < 0) {
  console.log("El número es negativo.");
}else {
  console.log("El número es neutro");
}
